### ISSUE TITLE
ExperimentHub with zero records

### ISSUE THREAD

**Issue by Pedramto89** [LTLA/scRNAseq#40]
Hi
I am trying to use SingleR for annotating my clusters. I would use "Pancreas" datasets but I get this error:
`> query(eh, "BaronPancreasData('human')")
ExperimentHub with 0 records
# snapshotDate(): 2022-10-31`

I tried to use other datasets for pancreas but still get the same error.
Would you help me?

**Comment by LTLA** [comment]
`BaronPancreasData('human')` is the function call, not the query to ExperimentHub. Just run it as a command.

**Comment by Pedramto89** [comment]
Thank you!
So you mean like this: "BaronPancreasData('human')"?
Then, when I want to make the query out of this dataset, how should I do it?
This is the workflow I follow:
`eh <- ExperimentHub()
query(eh, "BaronPancreasData('human')")`

**Comment by LTLA** [comment]
Yes, just run it directly. See `?BaronPancreasData` for more details.

**Comment by Pedramto89** [comment]
Thank you it worked. I thought it only has pancreas samples while it has everything other than pancreas. 
Here is the pipeline I followed:

eh <- ExperimentHub()
d <- BaronPancreasData('human')
query(eh, "d")

Then, we I run query() like above, it shows that:
**retrieve records with, e.g., 'object[["EH1"]]'**
I do this and run:
pdac_ref <- eh[["EH1"]]

In the pdac_ref object I created there is:
assayData: 23368 features, 7706 samples 

So, as I explained, I am looking for pancreas (tumor samples) which should be in the short from of "PAAD" or "PDAC" while in the metadata there is not:

levels(pdac_ref$CancerType)
 [1] "BLCA" "BRCA" "COAD" "GBM"  "HNSC" "KICH" "KIRC" "KIRP" "LAML" "LGG"  "LIHC" "LUAD"
[13] "LUSC" "OV"   "PRAD" "READ" "SKCM" "STAD" "THCA" "UCEC"

How can I extract the data only for pancreas?
Thanks!!

**Comment by LTLA** [comment]
I don't really know where to start here, because your comments suggest that you have a major misunderstanding of how programming works. `query(eh, "d")` doesn't make any sense. `eh[["EH1"]]` has no relevance to the **scRNAseq** package. Just run `BaronPancreasData()` and look at the output.

**Comment by Pedramto89** [comment]
Thank you for questioning my programming skills!!
